.gitignore

```diff
@@ -23,3 +23,4 @@ build/
 opensrp-path/fabric.properties
 opensrp-path/reports/
 opensrp-app/reports/
+opensrp-app/jacoco.exec
```

Comment:
Why is this removed. This file will be committed when running tests locally

Reply:
It's been ignored to avoid that very case